src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1447,7 +1446,7 @@ static bool processSubscriptions
     }
 
     /* Check 2: String Filter */
-    if (!trigs->stringFilter.match(notifyCerP))
+    if (trigs->stringFilterP && !trigs->stringFilterP->match(notifyCerP))
```

Comment:
use `if (() && ())`

(Who should do the change? @kzangeli is this your part or mine?)